source/docs/essentials/create-a-ui-component.md

```diff
@@ -0,0 +1,327 @@
+import { H3 } from "theme/ui/atoms/Typography/Heading";
+import Paragraph from "theme/ui/atoms/Typography/Paragraph";
+
+storiesOf("molecules.IllustratedContent", module)
+  .add("default", () => {
+    return (
+      <IllustratedContent media={<Icon icon="truck" />}>
+        <H3>Shipping within 48h</H3>
+      </IllustratedContent>
+    );
+  })
+  .add("with a lot of content", () => {
+    return (
+      <IllustratedContent media={<Icon icon="truck" />}>
+        <H3>Shipping within 48h</H3>
+        <Paragraph>
+          We are using many delivery services to let you choose what is best for
+          you!
+        </Paragraph>
+      </IllustratedContent>
+    );
+  })
+  .add("with an image", () => {
+    return (
+      <IllustratedContent
+        media={
+          <Image src="http://via.placeholder.com/350x150" alt="Placeholder" />
+        }
+      >
+        <H3>Shipping within 48h</H3>
+      </IllustratedContent>
+    );
+  });
+```
+
+It has many major benefits such as:
+
+- document edge cases
+- provide a test suite thanks to snapshot testing
+  ([Storyshots](https://github.com/storybooks/storybook/tree/master/addons/storyshots))
+- create a common discussion base for designers, product managers, marketers,
+  etc.
+
+[Learn more about Storybook.](#TODO)
+
+### Use the component
+
+Once you are satisfied with your component, you can use it anywhere. In this case,
+the `IllustratedContent` was to be used in the Reinsurance Banner. Thus, this
+module's component would look like this:
+
+```jsx
+import React from "react";
+import InlineCards from "theme/components/organisms/InlineCards";
+import IllustratedContent from "theme/components/molecules/IllustratedContent";
+import Icon from "theme/components/atoms/Icon";
+import { H3 } from "theme/components/atoms/Typography/Heading";
+
+// The components can then be used the usual React way
+
+const ReinsuranceBanner = () => (
+  <InlineCards>
+    <IllustratedContent media={<Icon icon="truck" />}>
+      <H3>Shipping within 48h</H3>
+    </IllustratedContent>
+    <IllustratedContent media={<Icon icon="thumbsup" />}>
+      <H3>Money back guarantee</H3>
+    </IllustratedContent>
+    <IllustratedContent media={<Icon icon="creditcard" />}>
+      <H3>Secured Payment</H3>
+    </IllustratedContent>
+  </InlineCards>
+);
+
+export default ReinsuranceBanner
+```
+
+As you can see, we did not use a relative import. This is because in Front-Commerce
+we have a few aliases that will let you import files without worrying about your
+current position in the folder structure.
+
+In our case, if the `ReinsuranceBanner` was in
+`my-module/web/theme/modules/ReinsuranceBanner`, we don't have to import the
+`IllustratedContent` by using relative paths
+`../../components/molecules/IllustratedContent` but we can remain with
+`theme/components/molecules/IllustratedContent` which is more explicit. This is
+possible for any file located in the folder `web/theme` of a module.
```

Comment:
```suggestion
possible for any file located in the folder `web/theme` of a module. <-- TODO Add a link to the aliases documentation page -->
```

I guess we will explain it more in depth at some point in the advanced section of the documentation.